qa/tasks/vstart_runner.py

```diff
@@ -1340,9 +1348,9 @@ def exec_test():
             p = remote.run(args=[os.path.join(BIN_PREFIX, "ceph"), "auth", "get-or-create", client_name,
                                  "osd", "allow rw",
                                  "mds", "allow",
-                                 "mon", "allow r"])
+                                 "mon", "allow r"], stdout=StringIO())
 
-            open("./keyring", "ab").write(p.stdout.getvalue())
+            open("./keyring", "ab").write(p.stdout.getvalue().encode())
```

Comment:
Could this one just use the BytesIO() instead ?

Reply:
In my personal opinion, we should use only one exclusively until there's hard requirement for otherwise to avoid confusion

Reply:
another way to address this is to open the keyring in `"at"` mode.

Reply:
Right. Made the changes. Thanks for pointing that out.

Reply:
@rishabh-d-dave You should miss Kefu's last comment here ? Let's `s/ab/at/` in open and get rid of the `encode()`.

Reply:
I guess I missed it while doing `git commit -p`. Thanks for reminding!

Reply:
Fixed. Thanks!